src/scopes.ts

```diff
@@ -478,6 +483,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     description: 'View the consent manager configuration.',
     products: [TranscendProduct.ConsentManager],
   },
+  [ScopeName.ViewAssessments]: {
+    title: 'View Assessments',
+    dependencies: [],
+    description: 'View the assessments and assessment templates.',
+    type: ScopeType.View,
+    products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
+  },
+  [ScopeName.ManageAssessments]: {
+    title: 'Manage Assessments',
+    dependencies: [ScopeName.ViewAssessments],
+    description: 'Manage and edit assessments and assessment templates',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
+  },
```

Comment:
Need to add here approveAssessments as well

Reply:
done